onadata/libs/utils/cache_tools.py

```diff
@@ -26,6 +26,31 @@
 XFORM_LINKED_DATAVIEWS = "xfs-linked_dataviews"
 PROJECT_LINKED_DATAVIEWS = "ps-project-linked_dataviews"
 
+# Cache names used in the open_data viewet
+METADATA_FIELDS = [
```

Comment:
I don't think these fields are being cached, perhaps move this to commono_tags.py?

Reply:
Moved